code/modules/mob/language/station_vr.dm

```diff
@@ -105,14 +105,21 @@
 
```

Comment:
//CHOMPAdd Start


all the way down to line 138 then there //CHOMPAdd End

Reply:
Already have a comment for it.